src/components/molecules/NavBar/NavBar.scss

```diff
@@ -243,19 +192,86 @@ $border-radius: 28px;
 }
 
 .NavBar {
-  &__home-icon {
+  &__home--icon {
```

Comment:
I'm not sure `--` is good here, icon isn't a state of the `__home` element, right? 
I'm assuming it is a `__home-icon` element, not `__home` element with the modifier (state) of `--icon`.

Reply:
The above will probably be an open question for few of the names in this file. Maybe a revisit is in order.

Reply:
fixed in 433dfe8
replaced double `-` with single `-`